shuup/admin/modules/settings/forms/system.py

```diff
@@ -72,8 +73,40 @@ class OrderSettingsForm(BaseSettingsForm):
         ),
         required=False,
     )
+    allow_anonymous_orders = forms.BooleanField(
+        label=_("Allow Anonymous Orders"),
+        help_text=_("Whether or not anonymous orders (without a `creator` user) are allowed."),
+        required=False,
+    )
 
 
 class OrderSettingsFormPart(BaseSettingsFormPart):
     form = OrderSettingsForm
     name = "order_settings"
+
+
+class CoreSettingsForm(BaseSettingsForm):
+    title = _("Core Settings")
+    home_currency = forms.CharField(
+        label=_("Home Currency"),
+        help_text=_(
+            "This option defines the currency in that all the monetary values are expressed. "
+            "Enter a valid ISO-4217 currency code."
+        ),
+        required=True,
+    )
+    address_home_country = forms.CharField(
```

Comment:
this can have the list of countries as options. we use django-countries, so you probably can use it here as well. maybe there is a form field ready for that